ironmon_tracker/Constants.lua

```diff
@@ -28,6 +28,7 @@ Constants.ButtonTypes = {
 	PIXELIMAGE = 6,
 	POKEMON_ICON = 7,
 	STAT_STAGE = 8,
+	CIRCLE = 9,
```

Comment:
Originally i built this Circle button as the clickable button on the main tracker screen, between Moves #/# and PP. I couldnt really get it to fit well, but i had put in the logic to draw it and all that. I figured i'd just leave the new button type in so it can be used later (not sure how to strip out code bits from a branch really). The drawing logic likely needs to be a bit better if we do intend to use this, however.